.github/workflows/coverage-main-branch.yml

```diff
@@ -32,7 +32,7 @@ jobs:
 
             - name: Prepare for Report Coverage (mini)
               run: |
-                  npx nyc report --reporter json-summary --exclude-after-remap false
```

Comment:
- [ ] please do the same in https://github.com/Accenture/sfmc-devtools/blob/cf07231ed0b0c8702405659875e8a6716311e9b3/.github/workflows/coverage-develop-branch.yml#L35